### PR TITLE
Add workflows for stable branches

### PR DESCRIPTION
**Pull request by aanm**
Since `issue_comment` is only triggered in GH workflows of the default
branch, we need to add a copy of the conformance GH workflows in the
default branch as well. This will allow the conformance tests to be
tested against stable branches, which right now only 1.10 has support
for this.

### CODE REVIEW

**Review by joestringer — APPROVED** [review]
Issue template makes sense. LMK if you want me to take a closer look at any workflows.

File: .github/workflows/conformance-aws-cni-v1.10.yaml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+          cilium version
+
+      - name: Install eksctl CLI
+        run: |
+          curl -LO "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz"
+          sudo tar xzvfC eksctl_$(uname -s)_amd64.tar.gz /usr/bin
+          rm eksctl_$(uname -s)_amd64.tar.gz
+
+      - name: Set up AWS CLI credentials
+        uses: aws-actions/configure-aws-credentials@13d241b293754004c80624b5567555c4a39ffbe3
```
Comment (nbusseneau) [inline .github/workflows/conformance-aws-cni-v1.10.yaml:172]: Nit: #16942
```suggestion
        uses: aws-actions/configure-aws-credentials@0d9a5be0dceea74e09396820e1e522ba4a110d2f
```

File: .github/workflows/conformance-eks-v1.10.yaml [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+          cilium version
+
+      - name: Install eksctl CLI
+        run: |
+          curl -LO "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz"
+          sudo tar xzvfC eksctl_$(uname -s)_amd64.tar.gz /usr/bin
+          rm eksctl_$(uname -s)_amd64.tar.gz
+
+      - name: Set up AWS CLI credentials
+        uses: aws-actions/configure-aws-credentials@13d241b293754004c80624b5567555c4a39ffbe3
```
Comment (nbusseneau) [inline .github/workflows/conformance-eks-v1.10.yaml:174]: Nit: #16942
```suggestion
        uses: aws-actions/configure-aws-credentials@0d9a5be0dceea74e09396820e1e522ba4a110d2f
```

**Review by nbusseneau — APPROVED** [review]
I'm very reluctant to go forward with having to maintain multiple versions of the same workflow but I don't have a better solution at the moment.

File: .github/workflows/conformance-aws-cni-v1.10.yaml (hunk 1 above)
Comment (aanm) [reply .github/workflows/conformance-aws-cni-v1.10.yaml:172]: Let's wait for the dependabot to do this automatically

**Pull request merged**